server/src/main/java/io/crate/protocols/postgres/types/CharacterType.java

```diff
@@ -75,7 +74,7 @@ public int writeAsText(ByteBuf buffer, @NotNull String value) {
 
     @Override
     protected byte[] encodeAsUTF8Text(@NotNull String value) {
-        return DataTypes.CHARACTER.implicitCast(value).getBytes(StandardCharsets.UTF_8);
+        return value.getBytes(StandardCharsets.UTF_8);
```

Comment:
The cast here was using the default instance of length = 1 and would truncate string to only one char, thus making `test_character_can_be_inserted_and_queried()` to fail.